cg/models/cg_config.py

```diff
@@ -301,27 +301,6 @@ class DataFlowConfig(BaseModel):
     archive_repository: str
 
 
-class PacbioConfig(BaseModel):
-    data_dir: str
-    systemd_trigger_dir: str
-
-
-class OxfordNanoporeConfig(BaseModel):
-    data_dir: str
-    systemd_trigger_dir: str
-
-
-class IlluminaConfig(BaseModel):
-    flow_cell_runs_dir: str
-    demultiplexed_runs_dir: str
-
-
-class RunInstruments(BaseModel):
-    pacbio: PacbioConfig
-    nanopore: OxfordNanoporeConfig
-    illumina: IlluminaConfig
-
-
```

Comment:
I think you need to check your branch. This undoes some changes introduced with other prs

Reply:
Yes I had a problem with git, I created another branch (Remove dryrun flag #3244) to replace this one.

Reply:
Feel free to close this one to avoid confusion :)